quick-start/index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link type="image/png" rel="icon" href="https://raw.githubusercontent.com/GMartigny/pencil.js/master/media/mini-logo.png">
+    <title>Quick-start - Pencil.js</title>
+    <link type="text/css" rel="stylesheet" href="../res/reset.css"/>
+    <link type="text/css" rel="stylesheet" href="../res/style.css"/>
+</head>
+<body>
+<div class="title-screen">
+    <div class="main-title">
+        <h1>
+            Quick-start
+        </h1>
+        <small>
+            Clear step-by-step demonstration of Pencil.js capabilities
+        </small>
+    </div>
+    <div class="element description">
+        <p>
+            Come along to create a nice animation in <strong>50 lines of codes</strong> in 9 steps.<br>
+            You can easily follow as each steps introduce <strong>only one</strong> feature at a time.<br>
+            This is the <strong>easiest way</strong> to get to know Pencil.js and should take about 5mn to read.
+        </p>
+        <div>
+            <a class="btn" href="../guide/">Guide</a>
+            <a class="btn" href="https://codepen.io/collection/XqzkNQ/">Examples</a>
+            <a class="btn" href="../enterprise/">For enterprises</a>
+        </div>
+    </div>
+</div>
+
+<!-- *** SETUP *** -->
+<div class="screen">
+    <h2>Setup</h2>
+    <p>
+        <a href="https://caniuse.com/#feat=mdn-javascript_statements_import">On capable browsers</a>,
+        the simplest way to use Pencil.js is to load it using the import statement.
+    </p>
+    <div class="example" data-script="n1"></div>
+    <p>
+        First thing you will need is a {Scene}. Once created, you can render it to display its content.
+    </p>
+    <div class="example" data-script="n2"></div>
+    <p>
+        Of course, there's nothing to render yet. Next, we will start to add {Component} and see the real power of the library.
```

Comment:
```suggestion
        Next, we will start to add a {Component} and see the real power of the library.
```
I would say that there is no room for "Of course" in a guide.. and it subtracts from the Signal-to-noise ratio.